_docs/conversions.md

```diff
@@ -0,0 +1,180 @@
+
+The basic workflow is that GitHub repos are created when posting a new tool. When files are posted or put to a version of a tool, we will create or delete and re-create a GitHub release/branch/tag with a matching name. When Dockerfiles are added, the tool will be created and built as a Quay.io repo. After adding both Dockerfiles and descriptors, you basically have a tool that is ready to be quickly registered and published under a Dockstore 1.2 web service. Go to Dockstore, do a refresh, and then hit quick register on the repos that you wish to publish. You can also do this programmatically through the write api client
+
+#### Client Prerequisites
+
+* Write API web service and all its prerequisites
+
+By now, then web service should be up and running with valid GitHub and Quay.io tokens. If not, please return to the web service usage section to get that running first. It is advised to ensure that the Write API web service is functioning correctly before using the client.
+
+* [Dockstore token](https://docs.dockstore.org/docs/publisher-tutorials/register-on-dockstore/)
+
+Follow the "Getting Started with Dockstore" tutorial to get a Dockstore token. Note this down, it will later be used in the Write API client properties file.
+
+* Dockstore server-url
+
+The Dockstore tutorial earlier would've specified the server-url alongside the token. Unless you're running your own dockstore webservice, the Dockstore production server-url is "https://www.dockstore.org:8443" and the Dockstore staging server-url is "https://staging.dockstore.org:8443". Note this down, it will also later be used in the Write API client properties file.
+
+* Quay.io integration
+
+In order to publish to Dockstore, Quay.io must be linked to Dockstore. See [Dockstore](https://docs.dockstore.org/docs/publisher-tutorials/register-on-dockstore/#linking-with-external-services) on how to link your Quay.io account to Dockstore.
+
+* Write API web service URL
+
+You will need to know the URL of the Write API web service you ran previously. If you've been using the example.yml, it should be "http://localhost:8082/api/ga4gh/v1"
+
+#### Client Configuration
+
+The configuration file used by the write-api-client is located at ~/.dockstore/write.api.config.properties
+It should look something like this:
+
+```
+dockstoreToken=abcdefghijklmnopqrstuvwxyz1234567890
+server-url=https://www.dockstore.org/api
+organization=test_organization
+repo=test_repository
+write-api-url=http://localhost:8080/api/ga4gh/v1
+```
+
+"dockstoreToken" is acquired from your account page on the dockstore website .
+"server-url" is the dockstore server url.
+"organization" is the organization/user of the repository to create.
+"repo" is the repository to create.
+"write-api-url" is the url of the write-api-service
+
+
+### Client Usage
+```
+Usage: client [options] [command] [command options]
```

Comment:
I don't think it is described how/where `client` comes from/how it gets installed.